packages/joint-core/src/dia/LinkView.mjs

```diff
@@ -816,6 +817,34 @@ export const LinkView = CellView.extend({
         return connectionPoint.round(this.decimalsRounding);
     },
 
+    isIntersecting: function(geometryShape, options) {
```

Comment:
As I understand, here it is different signature than in CellView (options here goes into 4th parameter of intersection call). I think it should has the same signature as CellView because it is effectively overrides parent method. And it is not reflected in types.

Reply:
It's actually ok.

```ts
return intersection.exists(
    connection,
    geometryShape,
    { segmentSubdivisions: this.getConnectionSubdivisions() },
    options
);
```

and

```ts
return intersection.exists(
    geometryShape,
    connection,
    options,
    { segmentSubdivisions: this.getConnectionSubdivisions() },
);
```

are equivalent.

Let me at least rename the `options` param to `geometryData` to match the parent class.